app/api/helpers/files.py

```diff
@@ -113,6 +114,8 @@ def create_save_resized_image(image_file, basewidth=None, maintain_aspect=None,
     uploaded_url = upload(upfile, upload_path)
     os.remove(temp_file_path)
 
+    uploaded_url = re.sub(r'\:(\d+)|None/', '/', uploaded_url)
```

Comment:
@iamareebjamal Is this the right place for stripping the port? I'm unsure as this is outside the test framework. Now the files fetched don't involve port number in their names.

Reply:
I think port shouldn't be stripped here at all. Because this is returning the url as well. It should be handled correctly when opening file. How is the url affecting the file name. This is the problem